README.md

```diff
@@ -7,13 +7,18 @@
 Helpful methods for exploring *in vivo* electrophysiology data.
 
 ## Installation
-To use the software, in the root directory, run
+
+### For users
+
 ```bash
-pip install -e .
+pip install aind-ephys-utils
 ```
 
-To develop the code, run
+### For developers
+
 ```bash
+git clone https://github.com/AllenNeuralDynamics/aind-ephys-utils
```

Comment:
Using the ssh git clone is recommended. Maybe just say clone the repository and in the root directory run `pip install -e .[dev]` or something like that.